WooCommerce/Classes/Tools/SiteCountry.swift

```diff
@@ -0,0 +1,664 @@
+        case GG
+        case GN
+        case GW
+        case GY
+
+        // H
+        case HT
+        case HM
+        case HN
+        case HK
+        case HU
+
+        // I
+        case IS
+        case IN
+        case ID
+        case IR
+        case IQ
+        case IE
+        case IM
+        case IL
+        case IT
+        case CI
+
+        // J
+        case JM
+        case JP
+        case JE
+        case JO
+
+        // K
+        case KZ
+        case KE
+        case KI
+        case KW
+        case KG
+
+        // L
+        case LA
+        case LV
+        case LB
+        case LS
+        case LR
+        case LY
+        case LI
+        case LT
+        case LU
+
+        // M
+        case MO
+        case MK
+        case MG
+        case MW
+        case MY
+        case MV
+        case ML
+        case MT
+        case MH
+        case MQ
+        case MR
+        case MU
+        case YT
+        case MX
+        case FM
+        case MD
+        case MC
+        case MN
+        case ME
+        case MS
+        case MA
+        case MZ
+        case MM
+
+        // N
+        case NA
+        case NR
+        case NP
+        case NL
+        case NC
+        case NZ
+        case NI
+        case NE
+        case NG
+        case NU
+        case NF
+        case KP
+        case MP
+        case NO
+
+        // O
+        case OM
+
+        // P
+        case PK
+        case PS
+        case PA
+        case PG
+        case PY
+        case PE
+        case PH
+        case PN
+        case PL
+        case PT
+        case PR
+
+        // Q
+        case QA
+
+        // R
+        case RE
+        case RO
+        case RU
+        case RW
+
+        // S
+        case ST
+        case BL
+        case SH
+        case KN
+        case LC
+        case SX
+        case MF
+        case PM
+        case VC
+        case WS
+        case SM
+        case SA
+        case SN
+        case RS
+        case SC
+        case SL
+        case SG
+        case SK
+        case SI
+        case SB
+        case SO
+        case ZA
+        case GS
+        case KR
+        case SS
+        case ES
+        case LK
+        case SD
+        case SR
+        case SJ
+        case SZ
+        case SE
+        case CH
+        case SY
+
+        // T
+        case TW
+        case TJ
+        case TZ
+        case TH
+        case TL
+        case TG
+        case TK
+        case TO
+        case TT
+        case TN
+        case TR
+        case TM
+        case TC
+        case TV
+
+        // U
+        case UG
+        case UA
+        case AE
+        case GB
+        case US
+        case UY
+        case UZ
+
+        // V
+        case VU
+        case VA
+        case VE
+        case VN
+
+        // W
+        case WF
+        case EH
+
+        // Y
+        case YE
+
+        // Z
+        case ZM
+        case ZW
+
+
+        var readableCountry: String {
+            switch self {
```

Comment:
I think the entire switch has to return localizable strings, right? So it should be

```
case .AX: return NSLocalizedString("Åland Islands", comment: "Returns the friendly, readable string for country code AX")
```

Reply:
Definitely. I forgot I had left this for the very end, thanks for pointing it out!